dev-tools/travis/print-errors-from-test-reports.py

```diff
@@ -55,6 +55,10 @@ def print_error_reports_from_report_file(file_path):
         if fail is not None:
             print_detail_information(testcase, fail)
 
+        failure = testcase.find("failure")
```

Comment:
@unsleepy22  the  api "find(match)" of Python ElementTree finds the first sub element matching match. So the match is sub element, but not sub string